projects/overlook.md

```diff
@@ -143,76 +166,118 @@ password: overlook2021
+</section>
 
-## Testing
+<section class="answer">
+### Fundamental JavaScript and Style / Fetch Competency Examples
+
+* Code is divided into logical components each with a clean, single responsibility
+* Array prototype methods are used to iterate instead of for loops
+* All DOM manipulation is held in the scripts.js file. No DOM manipulation occurs outside of this file.
+* Variables and functions are consistently and appropriately named
+* Code leverages JavaScript's truthy/falsey principles
+* Demonstrates efforts towards making functions pure when possible. **Note: Purity is not possible for every function in a FE application. Strive for it only when it makes sense.**
+* Implements GET and POST Fetch requests
+* When utilizing Fetch, the DOM is updated based on the results of that Fetch
+* Most errors are handled and messages are displayed to a user when an error occurs
+
+✨WOW✨ can look like: 
+
+* All data manipulation is cleanly separated from rendering code. 
+* Code has clearly been refactored; functions are DRY and adhere to the Single Responsibility Principle. 
+* All errors are handled and messages are displayed to a user when an error occurs
+* Effectively implements one or more closure throughout project.
+</section>
+
+<section class="answer">
+### Testing
+
+* Tests cover most features and test for happy paths
+* Test suite is organized.
+* Each function is tested in its own it block.
+* Each path is tested in its own it block.
+* Rather than using the production data, small sample data is stored in its own file and used for testing.
+* Sample data has been crafted to create the scenarios needed for thorough testing.
+* beforeEach hook is used to DRY up test files
+* There are no failing/pending tests upon submission 
+
+✨WOW✨ can look like: 
+
+* All scenarios/outcomes/paths are tested for functions, including happy and sad paths.
+* Application has a robust and thorough test suite that covers all functions that do not update the dom.
+</section>
+
+## Collaboration and Professionalism
 
-* 4: Application covers all aspects of the application including various flows and covers both happy/sad paths. Tests must be passing to be considered.
-* 3: Application is well tested but fails to cover some features and only tests for happy paths.  Tests must be passing to be considered.
-* 2: Project has sporadic use of tests at multiple levels. The application contains numerous holes in testing and/or many features are untested.  Tests must be passing to be considered.
-* 1: There is little or no evidence of testing in the application.
+See "Minimum Collaboration and Professionalism Expectations" above.
+While this is not a scored rubric section, every team member is expected to fully participate, contribute, communicate and collaborate with the team throughout the entirety of this project. Failure to do so can result in an individual failing the project, even if the group/project is otherwise passing.
```

Comment:
We should be able to get rid of this since this is always used a solo project :)

Reply:
Lol excellent point. I didn't even think about that when I threw this in 😂

Reply:
Updated!